examples/tutorial/jupyter/execution/pandas_on_ray/cluster/exercise_6.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
Maybe we should rename this file to cluster_exercise.py?

Reply:
I think the exercise number can help users understand in what order the exercises should be done because some of the tips from the tutorial for a single node can be useful for a cluster as well.